### PR TITLE
Add comment about overriding container_push destination

### PR DESCRIPTION
**Pull request by coryrc**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [na] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: container/push.bzl [hunk 1]
```diff
@@ -219,6 +219,9 @@ container_push_ = rule(
 )
 
 # Pushes a container image to a registry.
+# You can override some arguments by including adding arguments when running blaze run.
+# Additional arguments will be sent to the command listed in the _pusher rule above.
+# Most common use is adding --dst=myregistry/mypath:debugtag.
```
Comment (linzhp) [inline container/push.bzl:224]: Are you sure? I don't see `--dst` is a valid `bazel run` argument

Comment (coryrc) [reply container/push.bzl:224]: Need the `--` to pass arguments on to rule: `blaze run //target:image_push -- --dst=myregistry/mypath:debugtag`

Comment (linzhp) [reply container/push.bzl:224]: Can you include this example in the comment too? You can also comment that people can run `bazel run //target:image_push -- -help` to get full list of options

Comment (coryrc) [reply container/push.bzl:224]: Done

Comment (linzhp) [reply container/push.bzl:224]: I am not sure if blaze works with rules_docker, but I guess most people use bazel. Can you change it to "bazel"?

Comment (coryrc) [reply container/push.bzl:224]: d'oh, sorry

Comment (coryrc) [reply container/push.bzl:224]: and thank you for reviewing!

**Comment by coryrc** [comment]
@rbielak

**Review by rbielak — APPROVED** [review]

**Pull request merged**